trade-finance-manager-ui/templates/case/documents/documents.njk

```diff
@@ -1,6 +1,27 @@
+{% from "moj/components/banner/macro.njk" import mojBanner %}
+
 {% extends "case/case.njk" %}
 
 {% block sub_content %}
+  {% if hasAmendmentInProgress === true %}
```

Comment:
this for a future PR:
we can probably extract it inside a `_includes` file/folder or as a macro. This because the code repeats itself